server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -679,3 +680,13 @@ CREATE TABLE "ES_QUEUE" (
 );
 CREATE UNIQUE INDEX "PK_ES_QUEUE" ON "ES_QUEUE" ("UUID");
 CREATE INDEX "ES_QUEUE_CREATED_AT" ON "ES_QUEUE" ("CREATED_AT");
+
+CREATE TABLE "PLUGINS" (
+  "UUID" VARCHAR(40) NOT NULL PRIMARY KEY,
+  "KEE" VARCHAR(200) NOT NULL,
+  "BASE_PLUGIN_KEY" VARCHAR(200),
+  "JAR_MD5" VARCHAR(32) NOT NULL,
```

Comment:
would it be better to call it "hash" and have a longer max. lenght in case we want to change the algorithm at some point?